### DISCUSSION TITLE
Bug: "admin" is allowed on NamedTab type, but its value is ignored & NamedTab always included

### DISCUSSION THREAD

**Discussion by andrewjroberts** [payloadcms/payload#1284] [Q&A]
Hi Team Payload, 

It appears that while "admin" is allowed on a NamedTab type, that its value gets ignored.

If this is better meant as an issue, happy to delete and repost. 

---

**For reference:**

- NamedTab type below shows that "admin" property is included from union w/ FieldBase
  - `export declare type NamedTab = TabBase & FieldBase;`

- Here's a snapshot of what I'm trying to do — dynamically include the relevant tabs based on a enumerated layout property
  - <img width="654" alt="Screen Shot 2022-10-20 at 1 28 06 PM" src="https://user-images.githubusercontent.com/101012384/197018230-fac88810-72cb-4a18-9142-6e9dafc6a219.png">

- Results = still present even with always false value
  - <img width="259" alt="Screen Shot 2022-10-20 at 1 28 24 PM" src="https://user-images.githubusercontent.com/101012384/197018214-c0305ac0-8c0c-4398-8ca9-2162ce75bc71.png">

**Comment by PatrikKozak** [comment]
Hey @andwrobs — currently **conditions** are not meant to work on `tabs`. So we will need to omit that property from `FieldBase`. However, enabling it would be a good _feature request_ and we will keep it on the radar!
